net-mail/mu/mu-1.3.6.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools elisp-common
+
+DESCRIPTION="Set of tools to deal with Maildirs, in particular, searching and indexing"
+HOMEPAGE="http://www.djcbsoftware.nl/code/mu/ https://github.com/djcb/mu"
+SRC_URI="https://github.com/djcb/mu/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="emacs guile mug"
+
+# net-mail/mailutils also installes /usr/bin/mu.  Block it until somebody
+# really wants both installed at the same time.
+DEPEND="
+	dev-libs/gmime:3.0
+	>=dev-libs/xapian-1.4
+	dev-libs/glib:2
+	emacs? ( >=app-editors/emacs-24.4:* )
+	mug? ( >=net-libs/webkit-gtk-2.0 >=x11-libs/gtk+-3.0 )
```

Comment:
Also split this to multi-line, we are accustomed to reading one-dep-per-line so I originally missed gtk altogether. Both of these deps are slotted, you should depend on correct slot. The version requirement seems rather useless on both cases.